packages/react-core/src/demos/CardDemos.md

```diff
@@ -413,3 +416,622 @@ CardDetailsDemo = () => {
   );
 }
 ```
+
+### Aggregate status card
+
+```ts
+import * as React from 'react';
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  Divider,
+  Flex,
+  FlexItem,
+  Gallery,
+  Grid,
+  GridItem,
+  Stack
+} from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-triangle-icon';
+import TimesCircleIcon from '@patternfly/react-icons/dist/js/icons/times-circle-icon';
+
+const cardData = {
+  iconOnly: [
+    {
+      title: '5 Clusters',
+      content: [
+        {
+          icon: <CheckCircleIcon color="#3E8635" />
+        }
+      ],
+      layout: 'icon'
+    },
+    {
+      title: '15 Clusters',
+      content: [
+        {
+          icon: <ExclamationTriangleIcon color="#F0AB00" />
+        }
+      ],
+      layout: 'icon'
+    },
+    {
+      title: '3 Clusters',
+      content: [
+        {
+          icon: <TimesCircleIcon color="#C9190B" />
```

Comment:
> @mcarrano as we discussed, this is different from any of our other demos in that we're using custom CSS and utility classes to achieve a lot of what's in the demo. I'm wondering what a user's expectation is when referencing the demos and potentially wanting to use the code.
> 
> Utility classes don't ship with patternfly by default (you have to include them manually), and are typically large imports if you do include them, and the way they apply CSS is heavy-handed (using CSS's `!important`). If a user wants to copy and paste the code from the demo into their application, it's arguable that using utility classes isn't the best way to achieve a custom layout. Some users may know that, while others may not.
> 
> Should we denote somewhere in this demo that it's a departure from our other demos in that it uses custom CSS and utility classes?

Above is a quote the core issue, which is why I decided not to use utility classes. Is there a better way to show our consumers how to apply PF colors to icons?

Reply:
hm... interesting - because the core demo is using the utility classes...
But that's a good question for @mcoker or @mattnolting. I am pretty sure for theming purposes, and to prevent updates turning into breaking visual changes, we want to be sure we are utilizing the CSS API/variables/classnames correctly. So I think the ideal solution will be to add a class (whether a modifier, or a utility class), use a color variant if one exists, or update the value of a CSS variable?

Reply:
@nicolethoen @CooperRedhat that's right, we're using layouts and utilities here for most of the styles that might otherwise be custom CSS depending on how a product might want to build this page. Looks like the only custom CSS we have for these card demos is here, save a handful of inline `style` attributes for the gallery layout that the react gallery component will set via a prop - https://github.com/patternfly/patternfly/blob/main/src/patternfly/demos/Card/examples/Card.css#L18-L22.

> I am pretty sure for theming purposes, and to prevent updates turning into breaking visual changes

This is a great point and problem with using hex values like this. Just FYI, an alternative to using a hex value would be to use `--pf-global--warning-color--100`.